CMakeLists.txt

```diff
@@ -204,6 +204,7 @@ install(FILES ${COGCORE_API_HEADERS}
     DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}/cog
     COMPONENT "development"
 )
+file(COPY hooks/pre-commit DESTINATION ../.git/hooks)
```

Comment:
The relative path here is a bit concerning, I'm guessing this is relative to the build directory. We need this to be relative to the source directory - further up in the file, you can see `"$(CMAKE_SOURCE_DIR}/.git" referenced, we need to do the same here. In fact, this copy should probably go in that same block, we only ever want to do this for git checkouts and not source tarballs.

Reply:
It looks like squashing commits has brought this line back - the correct line is above, so this just needs removing